### DISCUSSION TITLE
Will Metamask extension support snaps?

### DISCUSSION THREAD

**Discussion by clostao** [MetaMask/snaps#1530] [Q&A] [answered]
Will Metamask extension support snaps or it will be exclusively for Metamask Flask?

If they will be supported there is some timeline defined for it?

**Comment by KillianH** [comment, marked as answer]
Check the pinned discussion here : https://github.com/MetaMask/snaps/discussions/1411

**Comment by Edumount** [comment]
> Check the pinned discussion here : #1411

thanks
